<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinConfig</class>
 <widget class="QDialog" name="WinConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>581</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>0</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>511</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="gbIns">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>161</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string>Vstupy</string>
    </property>
    <widget class="QComboBox" name="cbInputTypeSet">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsbInputDelaySet">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>440</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>9.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="pbInputTypeSet">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Nastav</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbInputDelaySet">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>470</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Nastav</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbOuts">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>201</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string>VÃ½stupy</string>
    </property>
    <widget class="QPushButton" name="pbOutputSafeSet">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>470</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Nastav</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbOutputTypeSet">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbOutputTypeSet">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Nastav</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsbOutputSafeSet">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>440</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>255.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbServos">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>201</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string>Serva</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WinConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WinConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
